5-Access-Authentication-and-Authorization/5.5-User-Accounts-and-Environment.bats

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env bats
+
+load 5.5.4-helper
+load 5.5.5-helper
+
+@test "5.5.1.1 Ensure minimum days between password changes is configured (Automated)" {
+    run bash -c "grep PASS_MIN_DAYS /etc/login.defs | grep --invert-match \"\#\""
+    [ "$status" -eq 0 ]
+    [[ "$output" != "" ]]
+    MINDAYS=(${output//PASS_MIN_DAYS/ }) # get the number from the string
+    [[ "$MINDAYS" -gt 0 ]]
+    run bash -c "grep -E ^[^:]+:[^\!*] /etc/shadow | cut -d: -f1,4"
+    [ "$status" -eq 0 ]
+    while IFS=: read -r line; do
+        MINDAYS=(${line//:/ }) # get the number from the string
+        [[ "${MINDAYS[1]}" -gt 0 ]]
+    done <<< "$output"
+}
+
+@test "5.5.1.2 Ensure password expiration is 365 days or less (Automated)" {
+    run bash -c "grep PASS_MAX_DAYS /etc/login.defs | grep --invert-match \"\#\""
+    [ "$status" -eq 0 ]
+    [[ "$output" != "" ]]
+    MAXDAYS=(${output//PASS_MAX_DAYS/ }) # get the number from the string
+    [[ "$MAXDAYS" -lt 366 ]]
+    run bash -c "grep -E '^[^:]+:[^!*]' /etc/shadow | cut -d: -f1,5"
```

Comment:
This command is not according to the CIS document (`awk -F: '(/^[^:]+:[^!*]/ && ($5>365 || $5~/([0-1]|-1|\s*)/)){print $1 " "
$5}' /etc/shadow`)